client/my-sites/plans-v2/selector-alt.tsx

```diff
@@ -143,40 +142,31 @@ const SelectorPageAlt = ( {
 		setDuration( selectedDuration );
 	};
 
-	const getProductsGrid = ( activeVariation = 'v0' ) => {
-		switch ( activeVariation ) {
-			case 'i5':
-				return ProductsGridAlt3;
-			case 'v2':
-				return ProductsGridAlt2;
-			default:
-				return ProductsGridAlt;
-		}
-	};
-
 	const viewTrackerPath = siteId ? `${ rootUrl }/:site` : rootUrl;
 	const viewTrackerProps = siteId ? { site: siteSlug } : {};
-	const currentABTestVariation = getJetpackCROActiveVersion();
-	const Grid = getProductsGrid( currentABTestVariation );
 
 	return (
 		<Main className="selector-alt__main" wideLayout>
 			<PageViewTracker path={ viewTrackerPath } properties={ viewTrackerProps } title="Plans" />
 
 			{ header }
 
-			<PlansFilterBar
-				showDiscountMessage
-				showDurations
-				onDurationChange={ trackDurationChange }
-				duration={ currentDuration }
-			/>
-
-			<Grid
-				duration={ currentDuration }
-				onSelectProduct={ selectProduct }
-				urlQueryArgs={ urlQueryArgs }
-			/>
+			{ showFilterBarInSelector() && (
+				<PlansFilterBar
+					showDiscountMessage
+					showDurations
+					onDurationChange={ trackDurationChange }
+					duration={ currentDuration }
+				/>
+			) }
```

Comment:
Just one thing I'm not understanding clearly.. What is the reasoning or advantage behind moving the `plansFilterBar` to the `Grid` component for the `i5` iteration? Will we conditionally (if the iteration is `i5`) pass down the `onDurationChange` function to the `Grid` component in this case?

Reply:
> What is the reasoning or advantage behind moving the plansFilterBar to the Grid component for the i5 iteration?

In i5, the filter bar is just below the section title and above the plan cards. I couldn't think of a better way to respect this layout other than doing this.

> Will we conditionally (if the iteration is i5) pass down the onDurationChange function to the Grid component in this case?

Most likely, yes.

Reply:
Oh yes, I see now. Makes sense. I didn't notice it is positioned below the bundle heading. Thanks for clarifying! 👍